app/invocation/invocation_model.tsx

```diff
@@ -215,10 +215,9 @@ export default class InvocationModel {
     for (let log of this.buildToolLogs?.log || []) {
       this.toolLogMap.set(log.name, new TextDecoder().decode(log.contents || new Uint8Array()));
     }
-    for (let commandLine of this.structuredCommandLine) {
-      if (commandLine.commandLineLabel != "canonical") {
-        continue;
-      }
+    let commandLine =
```

Comment:
Nit: I would appreciate it if there is a small comment explaining why we fall back to "original" and first event in the case of BuildBuddy Workflow.